### DISCUSSION TITLE
delay in updating the fields using setValue of useFieldArray

### DISCUSSION THREAD

**Discussion by JithenderSS** [react-hook-form/react-hook-form#9145] [Q&A]
I have an application where I'm trying to manage dropdown value as part of useFieldArray, when I'm trying to update the dropdown boolean using setValue, I'm able to toggle the dropdown. The problem is that when I have few fields there is no problem but when I have many fields, the dropdown value is updated with a delay. Is this the case only for me or should I follow any other approach to resolve this issue?

**Comment by leapful** [comment]
@JithenderSS : Could you please share codesandbox url that can re-produce the problem ?
